.github/workflows/check.yml

```
@@ -28,3 +28,30 @@ jobs:
       - uses: fredrikekre/runic-action@v1
         with:
           version: "1.4" # Keep version in sync with .pre-commit-config.yaml
+
+  explicit-imports:
+    runs-on: ubuntu-latest
+    name: "ExplicitImports.jl"
+    steps:
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v2
+        with:
+          version: 'nightly'
+      - uses: julia-actions/cache@v2
+      - uses: julia-actions/julia-buildpkg@v1
+      - name: Install dependencies
+        shell: julia {0}
+        run: |
+          # Add ExplicitImports.jl
+          using Pkg
+          Pkg.add([
+            PackageSpec(name = "ExplicitImports", version = "1.6"),
```

maybe it's a bug that has been fixed? not sure but 1.6 is pretty old
```suggestion
            PackageSpec(name = "ExplicitImports", version = "1.12"),
```